### ISSUE TITLE
deprecated supported features

### ISSUE THREAD

**Issue by dwdonnelly** [romedtino/simple-wyze-vac#77]
**Describe the bug**
The following is report in Home Assistant Logs:

Logger: homeassistant.helpers.entity
Source: helpers/entity.py:1567
First occurred: 6:44:52 AM (4 occurrences)
Last logged: 7:46:52 AM

Entity None (<class 'custom_components.simple_wyze_vac.vacuum.WyzeVac'>) is using deprecated supported features values which will be removed in HA Core 2025.1. Instead it should use <VacuumEntityFeature.STOP|RETURN_HOME|FAN_SPEED|BATTERY|STATUS|SEND_COMMAND|LOCATE|MAP|START: 11256>, please report it to the author of the 'simple_wyze_vac' custom integration and reference https://developers.home-assistant.io/blog/2023/12/28/support-feature-magic-numbers-deprecation

**To Reproduce**
N/A

**Expected behavior**
This msg to be resolved.

**Screenshots**
N/A

**Simple Wyze Vac Version**
1.8.9....That comes from the HACS web view.  

**Wyze Vacuum Firmware Version**
1.6.202  (I have two wyze vacuums, both are at the same version level) 

**Home Assistant Version**
Core  2024.2.1
Supervisor  2024.01.1
Operating System  11.5
Frontend  20240207.1

**Additional context**
Appreciate your help to correct this issue.

**Comment by tdavis75** [comment]
I can confirm that I'm seeing the same thing in my environment.

**Comment by dwdonnelly** [comment]
@tdavis75 Hopefully we will see a resolution from @romedtino.

**Comment by dwdonnelly** [comment]
@romedtino Appreciate any update on this issue.  Thanks.

**Comment by xStellarZ** [comment]
I'm not sure @romedtino will be fixing this as he hasn't been active in some time. I need this integration to work so I will fork and fix this if it breaks. @tdavis75 @dwdonnelly

**Comment by dwdonnelly** [comment]
@xStellarZ Many thanks.
@tdavis75 Four situational awareness

  As I am unfamiliar with the "fork" process, how does this work and where should I or we follow up on?  Many thanks in advance for your help.

**Comment by xStellarZ** [comment]
> @xStellarZ Many thanks. @tdavis75 For your situational awareness
> 
> As I am unfamiliar with the "fork" process, how does this work and where should I or we follow up on? Many thanks in advance for your help.

Went ahead and made a fork, will be updated here if/when I need to implement the changes: https://github.com/xStellarZ/simple-wyze-vac

**Comment by romedtino** [comment]
Hey, been quite busy with work but I wouldn't worry about that warning for now. We have until January'25 to make the changes and the entity changes aren't too hard to change but if anyone is interested in doing a PR of the change by all means 👍

**Comment by dwdonnelly** [comment]
Glad to hear the change should be easy and we have some time.  I appreciate the offer of the PR, but I am not technically skilled enough to attempt that.

 

For now, I will leave such things to the experts.

 

For planning purposes, when might you expect to work on this issue?  I don’t want to pester you.

 

Thanks !

Dave

**Comment by romedtino** [comment]
There's really no rush to this change and I don't know what kind of planning you'd need to do on your end since there's no changes on the user side of things. This is all just in the backend for the maintainer. With that said, a prerelease can be gotten through HACS - https://github.com/romedtino/simple-wyze-vac/releases/tag/1.9.0

**Comment by dwdonnelly** [comment]
Jerome,

  Understand.  Pre-Releases (please take no offense) always make me a little nervous – and as you point out, we have time.  I will be patient and wait.

Many thanks !

Dave

**Comment by dwdonnelly** [comment]
Jerome,

  New to github and it’s processes, so a couple of questions?

 

How do I know what version I have installed on my system?  The integration does not display.

 

If I understand this page, https://github.com/romedtino/simple-wyze-vac, V 1.9.0 has been released – no longer in a prerelease status.  Correct?

 

How do I update my installed version of Simple Wyze integration?  This would be done by deleting the current Simply Wyse integration and then going through the documented process to install the “new version”, as stated here: https://github.com/romedtino/simple-wyze-vac?tab=readme-ov-file#installation?  (there is no update button?  The update process is manual: delete/remove and reinstall?)

 

Many thanks for your help!

Dave
